apps/pubspec.yaml

```
@@ -26,23 +25,22 @@ dependencies:
   riverpod_lint: ^2.6.5
+  slang_build_runner: ^4.7.0
   yumemi_lints: ^4.1.1
 
 
 flutter:
-
-  uses-material-design: true
\ No newline at end of file
+  uses-material-design: true
+  assets:
+    - assets/i18n/
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)